proto/evmos/fees/v1/fees.proto

```diff
@@ -14,7 +14,3 @@ message DevFeeInfo {
   // it defaults to deployer_address
   string withdraw_address = 3;
 }
-
-// DevFeeInfosPerDeployer defines an instance that keeps track of all
-// contracts registered by a deployer
-message DevFeeInfosPerDeployer { repeated string contract_addresses = 1; }
```

Comment:
How can I remove a redundant proto message? CI is failing with
```
proto/evmos/fees/v1/fees.proto:1:1:Previously present message "DevFeeInfosPerDeployer" was deleted from file.
make: *** [Makefile:471: proto-check-breaking] Error 100
Error: Process completed with exit code 2.
```

Reply:
Q. Did you regenerate after removing?

Reply:
Yes. For example, this change is from regeneration https://github.com/tharsis/evmos/pull/586/files#diff-7435d9fe01ef2f1287c7f2c20c8a522585a6ce5755bfeb72f8ec552644c122deL91-L95

Reply:
See the `pb` files. They are regenerated.

Reply:
you can use `make proto-gen` to ignore the CI and build your proto files. We prob should update the proto-breakingness to compare it against the last stable version instead of `main`?

Reply:
Yes, I used `make proto-gen` with docker.
> should update the proto-breakingness to compare it against the last stable version instead of main

Comparing with the latest version sounds good to me. E.g. the fees module was not included in any version as far as I know.

Reply:
Sounds good, we can ignore the failing CI here and fix the proto breaking action in a follow-up